packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -179,17 +190,29 @@ $image-tile-text-transform: translate3d(5px, 0, 0);
     left: 0;
     right: 0;
     bottom: 0;
+
+    &:focus {
+        @include image-tile-focus();
+    }
 }
 
 .c-imageTile-link--toggle {
     display: block;
     position: static;
     pointer-events: none;
+
+    &:focus {
+        outline: none;
+    }
 }
 
 .c-imageTile-label {
     display: flex;
     flex-flow: column wrap;
+
+    &:focus {
+        outline: none;
+    }
 }
```

Comment:
Removing outline from the label to prevent Safari doubling focus styles.

Reply:
it might be worth adding this comment to code, because it's not obvious why this code was added.

Reply:
and this information might be useful if someone is refactoring this component

Reply:
Sure! I can add comments to both ✨

Reply:
thanks!